api/mock/facade.go

```diff
@@ -127,7 +127,7 @@ func (f *Facade) SendUserFunds(receiver string, value *big.Int) error {
 }
 
 // ExecuteSCQuery -
-func (f *Facade) ExecuteSCQuery(query *data.SCQuery) (*vmcommon.VMOutput, error) {
+func (f *Facade) ExecuteSCQuery(query *data.SCQuery) (*vm.VMOutputApi, error) {
```

Comment:
this is not VMOutputApi - it is a data structure. Suggestion to rename to WrappedVMOutput

Reply:
It somehow resembles with the other wrapping data structures used in API package, hence the suffix: `Api`

Reply:
Yes. In addition to what Iulian said, we have already merged the corresponding PR on elrond-go. Can remain like this